esmvalcore/_recipe.py

```diff
@@ -66,7 +66,6 @@ def read_recipe_file(filename, config_user, initialize_tasks=True):
 
 def _add_cmor_info(variable, override=False):
     """Add information from CMOR tables to variable."""
-    logger.debug("If not present: adding keys from CMOR table to %s", variable)
```

Comment:
Why do you reckon it's important? It didn't come across as very informative to me and I couldn't really think of a problem where this message would help with debugging.

Reply:
dunno actually, never seemed to be the crux for me either but I reckon it sounds important to a model developer that is running the tool on their newly developed model and they want to see what's happened to their variables? @jvegasbsc might be more in the know

Reply:
The only reason is it there is to keep track of when and how a key is added to the dataset, so a user will not wonder about why there are suddenly way more tags in a dataset than the ones defined in the recipe.

I am OK with it being removed to reduce the noise in the output, as I am not aware of any case in which this addition has been the source of a problem. It may potentially be if the tables are off for some reason, but after all our experience with the tool is a really low probability